src/Source/Adapter/FixedWidthFileInput.php

```diff
@@ -42,7 +43,7 @@ public function process($config)
 
             if (isset($config['condition'])) {
                 $condition = $config['condition'];
-                if (isset($condition['validIf']) && ! ConditionValidator::validateIsValid($condition['validIf'], $fieldsValues)) {
+                if (isset($condition['validIf']) && !ConditionValidator::validateIsValid($condition['validIf'], $fieldsValues)) {
```

Comment:
Avoid using static access to class '\Dealweb\Integrator\Validation\ConditionValidator' in method 'process'.